### PR TITLE
Expose trusted certificates support in helm starter template

### PR DESCRIPTION
**Pull request by CiprianAnton**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/install-systemlink-enterprise/blob/master/CONTRIBUTING.md).

TODO: Check the above box with an 'x' indicating you've read and followed [CONTRIBUTING.md](https://github.com/ni/install-systemlink-enterprise/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Expose options to specify trusted certificates in systemlink. 

### Why should this Pull Request be merged?

Custom certificates support was added for s3 communication.

### What testing has been done?

Tested on NI internal cluster.

### CODE REVIEW

**Review by zraul123 — APPROVED** [review]

File: getting-started/templates/systemlink-values.yaml
Comment (JD-Robertson) [inline getting-started/templates/systemlink-values.yaml:98]: Can we be clear in the comment that these are certificate authority public certificates which the user has hand-deployed as secrets on the cluster prior to installing SystemLink?

Comment (CiprianAnton) [reply getting-started/templates/systemlink-values.yaml:98]: Specified that the secrets must be deployed manually.

**Comment by CIakab-NI** [comment]
@CiprianAnton , We need Code Owners' approval for this PR for the SLE 2024-04 release:
@BKnight760 , would you be able to approve? 

I see JD has reviewed the changes and there was one more commit since.

**Review by JD-Robertson — APPROVED** [review]

**Pull request merged**
